routes.yml

```diff
@@ -0,0 +1,1502 @@
+services:
+  controller.benchmark.rest:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\RestController
+    arguments: [ "@serializer", "@event_dispatcher" ]
+  controller.benchmark.test-route-1:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-2:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-3:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-4:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-5:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-6:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-7:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-8:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-9:
+    public: true
+    class: PhpBenchmarksSymlex\Controller\TestController
+  controller.benchmark.test-route-10:
+    public: true0
```

Comment:
I will fix it after the merge

Reply:
Oops, I used a regex for replacing that - didn't seem to work for numbers > 9 :/